src/sentry/static/sentry/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -198,7 +202,8 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
       );
     }
 
-    return this.renderRequestIntegrationButton();
+    // This should never happen but we can't return undefined without some refactoring.
+    return <span />;
```

Comment:
I think we can return null instead. We don't want unnecessary dom elements.

Reply:
@NisanthanNanthakumar See the comment above. This line never will return anything (unless there is a bug). It's only there to make TS happy.

Reply:
@NisanthanNanthakumar I get what you're saying but the problem is the function signature. We don't want null to be an acceptable value to return. We can return an empty react fragment so we don't add dom elements.

Reply:
@scefali yeah sounds good to me!

Reply:
👍